### PR TITLE
Added student and class models

### PR DESCRIPTION
**Pull request by Asifkhan-akhan**

### CODE REVIEW

File: models/class.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package models
+
+import "github.com/fatih/structs"
+
+type Class struct {
+	ID   string `json:"id" structs:"id" bson:"_id" db:"id"`
+	Name string `json:"name" structs:"name" bson:"name" db:"name"`
+	//CourseID string `json:"courseID" structs:"courseID" bson:"courseID" db:"courseID"`
```
Comment (OvaisTariq95) [inline models/class.go]: remove this comment

Comment (OvaisTariq95) [inline models/class.go]: studentsEnrolled int

File: models/student.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"github.com/fatih/structs"
+)
+
+type Student struct {
+	ID      string  `json:"id" structs:"id" bson:"_id" db:"id"`
+	Name    string  `json:"name" structs:"name" bson:"name" db:"name"`
+	Age     string  `json:"age" structs:"age" bson:"age" db:"age"`
+	GPA     float64 `json:"gpa" structs:"gpa" bson:"gpa" db:"gpa"`
```
Comment (OvaisTariq95) [inline models/student.go]: CGPA

File: models/student.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	ClassID string  `json:"classID" structs:"classID" bson:"classID" db:"classID"`
```
Comment (OvaisTariq95) [inline models/student.go:12]: json:"classID" structs:"classID" bson:"classID" db:"classID
is this the correct json ?

**Review by OvaisTariq95 — CHANGES_REQUESTED** [review]
Good Work! we are close to wrapping this up

File: models/student.go (hunk 3 above)
Comment (Asifkhan-akhan) [reply models/student.go:12]: i think it is . it shows no error in test

**Pull request closed without merging**